challenges/exercises/ex43.py

```diff
@@ -0,0 +1,21 @@
+# Challenge 043
+
+"""
+Ask which direction the user wants to count (up or down).
+If they select up, then ask the for the top number and then count from 1 to that number.
+If they select down, ask them to enter a number below 20 and then count down from 20 to that number.
+If they entered something other than up or down, display the message "I don't understand."
+"""
+
+direction = input("Do you want to count up or down? up/down ")
```

Comment:
could use `.lower()` here in case someone capitalises any of the letters so it won't match either if statement